zkevm-circuits/src/bytecode_circuit/circuit.rs

```diff
@@ -803,14 +756,8 @@ impl<F: Field> BytecodeCircuitConfig<F> {
         region.name_column(|| "BYTECODE_q_first", self.q_first);
         region.name_column(|| "BYTECODE_q_last", self.q_last);
         region.name_column(|| "BYTECODE_length", self.length);
-        region.name_column(|| "BYTECODE_push_data_left", self.push_data_left);
```

Comment:
I think this line should not be removed, the `IsZeroChip.annotate_columns_in_region` only annotates the inverse column.